### PR TITLE
notebooks for plot_cumulative_gain and plot_lift_curve

### PR DESCRIPTION
**Pull request by cr458**
Added jupyter notebook examples for cumulative gain and lift curve.

### CODE REVIEW

File: examples/jupyter_notebooks/plot_cumulative_gain.ipynb [hunk 1]
```diff
@@ -0,0 +1,94 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### An example showing the plot_confusion_matrix method\n",
```
Comment (reiinakano) [inline examples/jupyter_notebooks/plot_cumulative_gain.ipynb]: Typo

Comment (cr458) [reply examples/jupyter_notebooks/plot_cumulative_gain.ipynb]: Oops! Fixed!

**Comment by reiinakano** [comment]
Thanks!!

**Comment by cr458** [comment]
np!

**Pull request merged**